.github/workflows/nightly.yml

```diff
@@ -0,0 +1,37 @@
+name: Nightly CI
+
+on:
+  schedule:
+    - cron: '11 7 * * 1,4'
+
+env:
+  RUSTFLAGS: -Dwarnings
```

Comment:
The rest of the `ci.yml` test suite does not deny warnings. Either add it there, too, or remove it here, please.